config/initializers/content_security_policy.rb

```diff
@@ -5,12 +5,12 @@
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
 Rails.application.config.content_security_policy do |policy|
-  policy.default_src :self, :https
+  policy.default_src :self
   policy.font_src :self, :https, :data
   policy.img_src :self, :https, :data
   policy.object_src :none
-  policy.script_src :self, :https
-  policy.style_src :self, :https
```

Comment:
What effect does dropping `:https` from these have?

Reply:
These lines define whitelists. The old version meant anything from `:self` (ie our domain) and anything over `https`. Removing `:https` means we no longer allow external styles or scripts.

Reply:
More info: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP

Reply:
Ah ok. I thought it was restricting to HTTPS only from self.